app/views/application_drafts/_student_fields.html.slim

```diff
@@ -8,8 +8,8 @@
   h2 Basics
   = s.input :name, label: 'Student name', hint: 'Please provide your first and last name.', input_html: { disabled: !may_edit?(s.object) }
   .form-inline
-    = s.input :application_age, :collection => User::AGE, include_blank: true, label: 'Your age', disabled: !may_edit?(s.object), wrapper_html: { class: student_field_error_class(index, application_draft, :application_age)}, hint: student_field_error_hint(index, application_draft, :application_age)
-  = s.input :application_gender_identification, as: :string, label: 'Your gender self-identification', input_html: { disabled: !may_edit?(s.object), "data-genders-src" => User::GENDER_LIST.to_json, class: "gender-form-field"}, wrapper_html: { class: student_field_error_class(index, application_draft, :application_gender_identification) }, hint: (student_field_error_hint(index, application_draft, :application_gender_identification) ||  'No-one is excluded from applying on the basis of gender, but people who self-identify as female, or have experiences being socialized as female, are given preference during selection.')
+    = s.input :application_age, :collection => StudentFields::AGE, include_blank: true, label: 'Your age', disabled: !may_edit?(s.object), wrapper_html: { class: student_field_error_class(index, application_draft, :application_age)}, hint: student_field_error_hint(index, application_draft, :application_age)
```

Comment:
Not sure how relevant for the moment, but I would vote for **not** decoupling the fields / constants from where they actually belong to and just reference them straight from the model.

So in this case we're dealing with an `ApplicationDraft` model / instance right? My initial idea would have been to simply include the fields module there
```ruby
class ApplicationDraft
  include Fields::StudentFields
```
This would add all the constants and functionality to the `ApplicationDraft` and you'd use it directly from there.